2024/door_12/index.js

```
@@ -0,0 +1,103 @@
+      const next = stars[index + 1];
+      if (next) next.focus();
+    } else if (e.key === 'ArrowLeft') {
+      const prev = stars[index - 1];
+      if (prev) prev.focus();
+    }
+  });
+});
+
+function setRating(rating) {
+  rating = parseInt(rating);
+  if (isNaN(rating) || rating < 1 || rating > 5) return;
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)